### DISCUSSION TITLE
Bots in single player with "Fight Intruder" command will not attempt to arrest any enemy. Kills handcuffed enemies.

### DISCUSSION THREAD

**Discussion by Argimor** [FakeFishGames/Barotrauma#15289] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In unmodded single player campaign(Client-side Lua, no mod packages running), any bot in the sub that has "Fight Intruder" command treats handcuffed human enemies as armed and dangerous, IE: runs away if unarmed, or attempts to kill, even with the means to arrest, IE: stun weapon and handcuff only loadout security officers. Tested with outpost pirates in normal campaign, prisoners who have been re-handcuffed after escaping, and any spawned human with any job type in "Team2" enemy team. Bots with the means to arrest do not attempt to do so, and will seek lethal weapons if they only have stun weapons, and will kill any enemy that is handcuffed.

Also It is difficult to tell but it appears normal AI not in "Fight Intruder" mode treats the handcuffed enemies differently, as if attempting to run away from them, handcuffed enemies also seem to react as if they are not handcuffed and instead only "weaponless" and try to run away too, additionally assaulting AI in "Fight Intruders" mode will treat temporarily stunned handcuffed enemies as neutralized, but the instant they get back up they attempt to kill, repeating until enemy is unconscious.

I'm fairly sure this is unintended behavior, as it makes it impossible to assign crew AI defense duties while capturing prisoners for the "By the Book" talent, or doing prisoner transport or any mission where a hostile NPC must be kept alive.

### Reproduction steps

1. Start single player campaign, or provided save, [Save.zip](https://github.com/user-attachments/files/17987879/Save.zip)
2. Assign any player crew AI with handcuffs and means to stun the "Fight Intruders" command and place in the submarine.
3. Spawn any hostile human NPC armed or not in submarine.
4. Crew AI will kill target, will not use handcuffs.
5. Spawn any hostile human NPC and handcuff them.
6. Crew AI will kill target.
7. Example:
![2024-12-02 23_16_22-Greenshot](https://github.com/user-attachments/assets/b5f33e88-9203-4296-b400-d80797085675)
![2024-12-02 23_18_20-Greenshot](https://github.com/user-attachments/assets/970a7ad0-9fc1-4de0-b1f3-3d18c4a38180)
 Crew continues fighting stunned target, does not attempt to handcuff.
![2024-12-02 23_19_04-Greenshot](https://github.com/user-attachments/assets/369afe5a-459d-45bb-8a32-385d73cb6fbe)
Manually handcuffing enemy.
![2024-12-02 23_19_46-Greenshot](https://github.com/user-attachments/assets/b4af86c0-f82f-4dcc-a38e-da5e46aa6efc)
"Fight Intruders" order cancelled, handcuffed enemy attempts to run away.
![2024-12-02 23_20_06-Greenshot](https://github.com/user-attachments/assets/d84a4f22-e06e-4c43-a568-e98eb8f9a1d5)
On resuming command, crew AI kills treats handcuffed enemy as hostile and kills them
![2024-12-02 23_20_22-Greenshot](https://github.com/user-attachments/assets/d3cea392-f923-42ae-89ff-1c72f49eb188)
![2024-12-02 23_20_37-Greenshot](https://github.com/user-attachments/assets/914dd5f5-1685-42bb-86cc-80fd0e99d009)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
